tests/unit/ui/components/TestRatingManagerMessage.js

```diff
@@ -0,0 +1,45 @@
+import { shallow } from 'enzyme';
```

Comment:
This test file needs to move to ` tests/unit/amo/components/TestRatingManagerMessage.js`. I missed that in the original patch.